pkg/dashboard/subproc/data.go

```diff
@@ -478,6 +479,14 @@ func (d *DataLayer) ChartRepoDelete(name string) (string, error) {
 	return out, nil
 }
 
+func (d *DataLayer) GetNameSpaces() (string, error) {
+	out, err := d.runCommandKubectl("get", "namespaces", "-o", "json")
```

Comment:
This method should return `[]string`, to keep data layer as a source of structured information. The web handlers layer should then decide if it's serialized as string/JSON/etc.

This is important because the methods of data layer are sometimes used internally. Also, it is more elegant in terms of code design.

Reply:
Actually the response from this `get namespace` command returns a json object. Did you mean it should be serialized into a struct ?

Reply:
yes, it should be _unserialized_ into struct

Reply:
updated